db/seeds.rb

```diff
@@ -1,7 +1,31 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.destroy_all
+Team.destroy_all
+TeamsUser.destroy_all
+Match.destroy_all
+
+user_names = ['jack', 'jan', 'den', 'rob']
+users      = []
+teams      = []
+
+4.times do |i|
+  users << User.create(name: "user-#{user_names[i-1]}", email: "example-#{user_names[i-1]}@mail.com",
+                       password: '123456', password_confirmation: '123456', username: "player-#{i}")
```

Comment:
maybe you want better user names that player#1 for when you will be adding them to a match for example through the interface